pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -266,3 +266,27 @@ def is_target_flashed_same_elf(self) -> bool:
             elf_sha256.update(fr.read())
 
         return flash_elf_sha256 == elf_sha256.digest()
+
+
+class LinuxSerial(DuplicateStdoutPopen):
+    """
+    Linux serial Dut class
+    """
+
+    def __init__(
+        self,
+        app: IdfApp,
+        msg_queue: MessageQueue,
+    ) -> None:
+        self.app = app
+        self.msg_queue = msg_queue
+
+        if not hasattr(self.app, 'target'):
+            raise ValueError(f'Idf app not parsable. Please check if it\'s valid: {self.app.binary_path}')
+
+        if self.app.target != 'linux':
+            raise ValueError(f'Targets do not match. App target: {self.app.target}, Cmd target: "linux".')
+
+        logging.info(f'executing {self.app.elf_file}')
+
+        super().__init__(msg_queue, self.app.elf_file)
```

Comment:
```suggestion
class LinuxSerial(DuplicateStdoutPopen):
    """
    Linux serial Dut class
    """

    def __init__(
        self,
        app: IdfApp,
        **kwargs
    ) -> None:
        self.app = app

        if not hasattr(self.app, 'target'):
            raise ValueError(f'Idf app not parsable. Please check if it\'s valid: {self.app.binary_path}')

        if self.app.target != 'linux':
            raise ValueError(f'Targets do not match. App target: {self.app.target}, Cmd target: "linux".')
        
        logging.info(f'executing {self.app.elf_file}')
        
        super().__init__(cmd=self.app.elf_file, **kwargs)
```

here I prefer to use the kwargs method. It would be easier for future maintenance.